src/main/java/org/broadinstitute/consent/http/db/mapper/InstitutionWithUsersReducer.java

```diff
@@ -58,6 +58,15 @@ public void accumulate(Map<Integer, Institution> map, RowView rowView) {
 
     institution.setCreateUser(create_user);
     institution.setUpdateUser(update_user);
+
+    if (Objects.nonNull(rowView.getColumn("so_dacuserid", Integer.class))) {
+      User so_user = new User();
+      so_user.setDacUserId(rowView.getColumn("so_dacuserid", Integer.class));
+      so_user.setEmail(rowView.getColumn("so_email", String.class));
+      so_user.setDisplayName(rowView.getColumn("so_displayname", String.class));
```

Comment:
Would be safer to have column checks on `so_email` and `so_displayname` as well. 

Another thought ... if we used `SimplifiedUser` here instead of `User`, then added a BeanMapper for that class on the DAO method, we could simplify this and let jdbi handle the class construction from whatever row values existed.